api/wasm/cpp/proxy_wasm_intrinsics.pb.cc

```diff
@@ -0,0 +1,364 @@
+// Generated by the protocol buffer compiler. DO NOT EDIT!
```

Comment:
Why is generated files are checked in?

Reply:
Because these are part of the WASM api which would be copied by users instead of used as part of envoy with bazel.  In the future when we sort out the tool chain we may do this differently, but it was expedient until we sort that.